src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -0,0 +1,24 @@
+module Workflows
+  class YAMLToWorkflowsService
```

Comment:
It would be nice if we can have a better name for this. Two times workflow and having service in the name is redundant.